components/tools/OmeroPy/test/integration/gatewaytest/test_get_objects.py

```diff
@@ -607,34 +626,35 @@ def testListOrphans(self, gatewaywrapper):
         # Pagination, loading pixels
         params = omero.sys.ParametersI()
         params.page(1, 3)
-        findImagesInPage = list(gatewaywrapper.gateway.listOrphans(
+        findImagesInPage = list(conn.listOrphans(
             "Image", eid=eid, params=params, loadPixels=True))
         assert len(findImagesInPage) == 3, \
             "Did not find orphaned images in page"
         for p in findImagesInPage:
             assert p._obj.pixelsLoaded
 
-        # All orphans, no pixels
-        findImages = list(gatewaywrapper.gateway.listOrphans("Image"))
+        # All orphans, without loading pixels
```

Comment:
Could the test be split up or parametrised? one test with ``loadPixels=True``, one ``False``